notebooks/GPU.ipynb

```diff
@@ -1,6 +1,6 @@
 {
   "cells": [
-    {
+	{
```

Comment:
Minor nitpick: unnecessary whitespace change

Reply:
Fuck, I hate whitespace changes. Four spaces instead of tab?

Reply:
Yep, it was originally 4 spaces and seems to have been replaced by a tab character.